packages/analytics/metric-cards/src/utilities/trend-display.ts

```diff
@@ -51,14 +53,14 @@ export const calculateChange = (curr: number, prev: number) => {
 /**
  * Determines whether to display an upward or downward trend, or no change
  */
-export const defineIcon = (polarity: number, thisIsBad: boolean = false): string => {
+export const defineIcon = (polarity: number, thisIsBad: boolean = false) => {
```

Comment:
```suggestion
export const defineIcon = (polarity: number, thisIsBad: boolean = false): TrendUpIcon | TrendDownIcon | IndeterminateSmallIcon => {
```

Reply:
You may have to do `typeof TrendUpIcon | typeof TrendDownIcon | typeof IndeterminateSmallIcon` since they are not types

Reply:
Last commit did the trick. Thanks!